ztrouy/frontend-capstone-api
# Feature - Username Editing

### Purpose
To allow Users to change their own Username

### Files Changed
- Added serializer and method for updating a Users' username in `users.py`

### To Test
Fetch, setup by running `pipenv shell` and `./seed_database.sh` in the project root directory, run in the debugger, then confirm the following
- PUT request `/users/{id}` updates the username of the appropriate User when provided with valid JSON, so long as the User is only attempting to edit their own username

closes #21
